src/t8_forest/t8_forest_cxx.cxx

```diff
@@ -398,7 +398,7 @@ t8_forest_element_from_ref_coords (t8_forest_t forest, t8_locidx_t ltreeid, cons
   double tree_ref_coords[3] = { 0 };
   const t8_eclass_t tree_class = t8_forest_get_tree_class (forest, ltreeid);
   const t8_eclass_scheme_c *scheme = t8_forest_get_eclass_scheme (forest, tree_class);
-  scheme->t8_element_reference_coords (element, ref_coords, NULL, tree_ref_coords);
+  scheme->t8_element_reference_coords (element, ref_coords, 1, tree_ref_coords);
```

Comment:
Please write down why this was changed to 1 from NULL

Reply:
This has changed, because the argument of the function itself changed. Before, the argument was for user data, which was not used at all and now it is for the amount of points to evaluate